go.sum

```diff
@@ -118,8 +118,10 @@ github.com/hdac-io/casperlabs-ee-grpc-go-util v0.7.0/go.mod h1:0T62kfiEVNVgdUJuC
 github.com/hdac-io/iavl v0.0.0-20191220074654-211e1bd34eef h1:2Agvbgr7rAy4OwBETZ6a4VAXLXCbPDdLWw8Mz9zQcLw=
 github.com/hdac-io/iavl v0.0.0-20191220074654-211e1bd34eef/go.mod h1:ry8RH8MNJJhiO1VJeosYAsKrPu7X8Xy0B5D5UQvEiVM=
 github.com/hdac-io/tendermint v0.0.0-20191220055750-d875915aea37/go.mod h1:BronDynhlemRSl99OCZ3owAs/cTcDuw0BrgFmzasWPk=
-github.com/hdac-io/tendermint v0.32.8-0.20200507142637-8c1454912aa4 h1:kzcOknvwrS9MGUAYefZBNCi48x9F8/c13WJU1ihxNFo=
-github.com/hdac-io/tendermint v0.32.8-0.20200507142637-8c1454912aa4/go.mod h1:lQzldj9uvA4kkM+GH9Ty/bXNzAl9wWJEAa8iKt0+bMQ=
+github.com/hdac-io/tendermint v0.32.8-0.20200510224311-bc74d177bea9 h1:cxlEGdlboScFGYKj/m6KSPvlFOqYcdwZ0f94Xd3kyhk=
+github.com/hdac-io/tendermint v0.32.8-0.20200510224311-bc74d177bea9/go.mod h1:lQzldj9uvA4kkM+GH9Ty/bXNzAl9wWJEAa8iKt0+bMQ=
+github.com/hdac-io/tendermint v0.32.8-0.20200512022919-d1fd21afc393 h1:QcvwUc4k0phg+lwyh6XMF6LLbeST7XkjvwUkGP8wk4M=
+github.com/hdac-io/tendermint v0.32.8-0.20200512022919-d1fd21afc393/go.mod h1:lQzldj9uvA4kkM+GH9Ty/bXNzAl9wWJEAa8iKt0+bMQ=
```

Comment:
The versions seem to be mixed. `Need to go mod tidy`

Reply:
The cause was the old tendermint connected to the iavl. Added fix commit.